fuzz.go

```diff
@@ -2,6 +2,7 @@
 // Copyright 2020 uint256 Authors
 // SPDX-License-Identifier: BSD-3-Clause
 
+//go:build gofuzz
```

Comment:
What is this?

Reply:
go 1.17 build directives have changed. go formt auto-converts the old style to the new one